test/f.jl

```diff
@@ -0,0 +1,35 @@
+using HypothesisTests, Test
+using HypothesisTests: default_tail
+
+@testset "F-tests" begin
+    @testset "Basic variance F-test" begin
+        Random.seed!(12)
+        y1_h0 = 4 .+ randn(500)
+        y2_h0 = 4 .+ randn(400)
+
+        t = VarianceFTest(y1_h0, y2_h0)
+
+        @test t.n_x == 500
+        @test t.n_y == 400
+        @test t.df_x == 499
+        @test t.df_y == 399
+        @test t.F ≈ 0.974693 rtol = 1e-5
+        @test pvalue(t) ≈ 0.784563 rtol = 1e-5
+        @test default_tail(t) == :both
```

Comment:
Can you test the two tails for the p-value here too?

Reply:
How did you get the ground truth?

Reply:
I have compared with `var.test` in R for validation.